brain/validator/validate_password.py

```diff
@@ -0,0 +1,130 @@
+'''@validate_password
+
+This file contains functions for validating the user's password against
+a set of requirements.
+
+'''
+
+from flask import current_app
+import yaml
+
+
+def load_min(app=True, root='/vagrant'):
+    '''@load_min
+
+    This method returns the minimum character requirement for passwords.
+
+    @app, indicates if function is to be used by application, or manually
+
+    '''
+    if app:
+        min_c = current_app.config.get('PASSWORD_MIN_C', None)
+        return {'password_min_c': min_c, 'error': None}
+    else:
+        with open(root + "/hiera/settings.yaml", 'r') as stream:
+            try:
+                min_c = \
+                       yaml.load(stream)['validate_password']['password_min_c']
+                return {'password_min_c': min_c, 'error': None}
+            except yaml.YAMLError as error:
+                return {'password_min_c': None, 'error': error}
+
+
+def load_max(app=True, root='/vagrant'):
+    '''@load_max
+
+    This method returns the maximum character requirement for passwords.
+
+    @app, indicates if function is to be used by application, or manually
+
+    '''
+    if app:
+        max_c = current_app.config.get('PASSWORD_MAX_C', None)
+        return {'password_max_c': max_c, 'error': None}
+    else:
+        with open(root + "/hiera/settings.yaml", 'r') as stream:
+            try:
+                max_c = \

```

Comment:
For readability, I would change the following:

```python
                min_c = \
                       yaml.load(stream)['validate_password']['password_max_c']
```

to the following:

```python
                yamlres = yaml.load(stream)
                min_c =yamlres['validate_password']['password_max_c']
```

**Note:** the above change is consistent with your previous [`pytest_crypto.py`](https://github.com/jeff1evesque/machine-learning/blob/master/test/manual/authentication/pytest_crypto.py#L14).  Also, it's still taking up the same number of lines of code.